### ISSUE TITLE
adapt_framework: a11y focus isn't set to unfocusable elements

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#3022]
### Subject of the issue
Elements which can normally not receive focus should be forced focus.
This currently only works if preventScroll is set on the focus call.
It should work without the preventScroll flag being true.

**Comment by moloko** [comment]
to replicate issue, click on any component in PLP then check the value of `document.activeElement` - it should be the 'heading' div of the component but instead ends up being either the PLP button (Firefox) or the `body` element (Chrome)
